test/utils/nodes/baseload.go

```diff
@@ -48,31 +49,13 @@ func GetLoad(k8sCli *kubernetes.Clientset, nodeName string) (Load, error) {
 
 	cpu := &resource.Quantity{}
 	mem := &resource.Quantity{}
-
 	for _, pod := range pods.Items {
-		// TODO: we assume a steady state - aka we ignore InitContainers
-		for _, cnt := range pod.Spec.Containers {
-			for resName, resQty := range cnt.Resources.Requests {
-				switch resName {
-				case corev1.ResourceCPU:
-					cpu.Add(resQty)
-				case corev1.ResourceMemory:
-					mem.Add(resQty)
-				}
-			}
-		}
+		req, _ := resourcehelper.PodRequestsAndLimits(&pod)
```

Comment:
let's handle the error, especially if unlikely

Reply:
`PodRequestsAndLimits` returns requests and limits, in our case we don't care about the limits

Reply:
my bad, I should have checked the function signature. In this case it is fine as it is indeed.